### PR TITLE
Added toggle to hide web content pages from main navigation with chec…

### PR DESCRIPTION
**Pull request by bc-adamferenzi**
Added checkbox to theme editor to toggle on/off web content pages in main navigation, which is a common customer request.

https://jira.bigcommerce.com/browse/MERC-1239

### CODE REVIEW

File: templates/components/common/navigation-menu.html [hunk 1]
```diff
@@ -8,11 +8,13 @@
                 {{> components/common/navigation-list}}
             </li>
         {{/each}}
+        {{#if theme_settings.show_content_navigation}}
```
Comment (mcampa) [inline templates/components/common/navigation-menu.html]: Please indent properly the inside content

File: config.json [hunk 2]
```diff
@@ -192,7 +192,8 @@
     "loadingOverlay-backgroundColor": "#ffffff",
     "pace-progress-backgroundColor": "#989898",
     "spinner-borderColor-dark": "#989898",
-    "spinner-borderColor-light": "#ffffff"
+    "spinner-borderColor-light": "#ffffff",
+    "show_content_navigation": true
```
Comment (mcampa) [inline config.json]: The problem with this is that default values is `true` so when merchants get the update they won't have this setting in their config and therefore the pages will disappear.

What you should do instead is, name this `hide_content_navigation` and inverse the logic

**Comment by mcampa** [comment]
👍

**Comment by mattolson** [comment]
Can you remove `npm-debug.log`?

**Comment by bc-adamferenzi** [comment]
Thanks @mattolson, I missed that it landed in there.  I've removed and resubmitteded

**Comment by mcampa** [comment]
Solve conflicts to merge

**Comment by bc-adamferenzi** [comment]
@mcampa apologies, I'm not too familiar with this, but is anything needed other than updating these two files since they're out of date/sync https://github.com/bigcommerce/stencil?

**Comment by mcampa** [comment]
You'll need to rebase your branch against master, solve the conflicts and then force push to your branch again.

**Comment by bc-adamferenzi** [comment]
@mcampa I botched this something awful getting my remotes out of order and it was more trouble to fix than to just replace :/  Sorry for the trouble, but I've moved it here: https://github.com/bigcommerce/stencil/pull/871

**Pull request closed without merging**